core/domain/exp_services_test.py

```diff
@@ -1198,7 +1198,7 @@ class ZipFileExportUnitTests(ExplorationServicesUnitTests):
 states_schema_version: %d
 tags: []
 title: A title
-""" % (
+""" %(
```

Comment:
There shouldn't be any changes here too. Similarly below.

Reply:
Thanks, I will correct it.